Makefile

```diff
@@ -1,44 +1,53 @@
 # Put targets here if there is a risk that a target name might conflict with a filename.
 # this list is probably overkill right now.
 # See: https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html
-.PHONY: test test-unit test-e2e images run format verify
+.PHONY: test test-unit test-e2e images run format verify help
```

Comment:
do we think we'll have a file called `help` ?

Reply:
very probably not :)

Reply:
so then I think you should remove help from `.PHONY` since that is the purpose (filename conflict)... no?

Reply:
fixed!